apps/src/templates/certificates/Certificate.jsx

```diff
@@ -146,7 +150,11 @@ function Certificate(props) {
           linkText={i18n.backToActivity()}
         />
       )}
-      <div id="uitest-certificate" style={certificateStyle}>
+      <div
+        id="uitest-certificate"
+        className={wrapperClassName}
```

Comment:
This might be an ignorant question, but I want to make sure it won't be a problem.

If the className attribute is undefined, nothing would crash right?

Reply:
correct. here's how it looks in the DOM without the experiment: 
![Screen Shot 2022-09-19 at 12 52 00 PM](https://user-images.githubusercontent.com/8001765/191104436-92663bb3-d9fe-4df2-8bda-619d8846a411.png)

Reply:
Thanks for the review!